### PR TITLE
Bubble Sort Algortithm: Requesting Code Review

### PR DESCRIPTION
**Pull request by edieatha**
## Milestones Bubble Sort Algorithm
* Used a for loop to iterate over the array elements. 
* Used yield inside your method definition.
* The block has two arguments that represent the two elements of the array that are currently being compared.
* The block’s return is similar to the spaceship operator.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# frozen_string_literal: true
+
+#
+# Bubble sort and Bubble sort by
+#
+def bubble_sort(array)
+  for i in (0...array.length - 1)
+    for i in (0...array.length - 1)
+      array[i], array[i + 1] = array[i + 1], array[i] if array[i] < array[i + 1]
```
Comment (shshamim63) [inline main.rb]: The default sort provides a sorter order in increasing order. But the method you have generated provides decreasing order. Please convert it as returns increasing order sorted array.

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## Status :traffic_light: Required Changes :tractor: 
___
Hi @edieatha and @mcervantes71 :wave: 
Awesome job :clap: Just two more cases you need to follow to declare this project as approved. Let's get to them.
___
### :spades: Mandatory Changes :spades: 
___
1. Your bubble_sort returns a sorted array which is in decreasing order. Check out the comment below.
2. Stickler is informing some errors. Please fix them.
___
##### Please fix the above-mentioned issues and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Comment by martin-cervantes1** [comment]
:heavy_check_mark: Change to increasing order
:heavy_check_mark: Fix stickler error

**Review by nikasulo — APPROVED** [review]
# Great work! @edieatha and @mcervantes71 
Please merge and keep moving forward! I wish you the very best and see you at the top! :rocket: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**